conu/backend/k8s/backend.py

```diff
@@ -55,6 +57,10 @@ def __init__(self, logging_level=logging.INFO, logging_kwargs=None):
         self.core_api = get_core_api()
         self.apps_api = get_apps_api()
 
+        self.namespaces = []
```

Comment:
nit: can we rename this to `managed_namespaces` so it's clear what this variables holds?